Dockerfile

```diff
@@ -9,6 +9,7 @@
 # unzip our binaries to. For reference:
 # https://access.redhat.com/containers/?tab=support#/registry.access.redhat.com/ubi8/ubi
 ####################
+# hadolint ignore=DL3006
```

Comment:
If folks are interested, hadolint rules are documented in  their wiki: https://github.com/hadolint/hadolint/wiki